src/components/MessageItem/MessageItem.jsx

```diff
@@ -57,7 +67,10 @@ const MessageItem = ({
   return (
     <MessageItemStyled $isShownAvatar={isShownAvatar}>
       {messageTypeText && userId && isShownAvatar && (
-        <LetterAvatarStyled onClick={handleOpen}>
+        <LetterAvatarStyled
+          $isCurrentUser={isCurrentUser}
+          onClick={isCurrentUser ? null : handleOpen}
```

Comment:
Can we try do this `{!isCurrentUser && handleOpen}`. I`m not sure it will work))

Reply:
done